pkg/hintrunner/hinter/hinter.go

```diff
@@ -185,13 +185,3 @@ func (sdm *SquashedDictionaryManager) PopIndex() (uint64, error) {
 	sdm.KeyToIndices[key] = indices[:len(indices)-1]
 	return index, nil
 }
-
-// Global context to keep track of different results across different
-// hints execution.
-type HintRunnerContext struct {
```

Comment:
Maybe for clarity, whe should have scope related functionalities in scope.go, dictionaries related functionalities in dict.go and keep the general HintRunnerContext in another file as an "interface"

Reply:
Sounds good. Then, I'll move all dictionary utility to a `dict.go` file and  all `HintRunnerContext` related code back to the `hinter.go` file